e2etests/orchestration/create-single-instance-test.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Copyright (C) 2024 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+usage="usage: $0 -d build_id -t target"
+
+build_id=
+target=
```

Comment:
empty `build_id` and `target` values can be send to the host orchestrator in the create request, so we can let the test run and fail in this case rather than having an early exit.

Reply:
Acknowledged. Sounds good.